ext/hal/nxp/mcux/drivers/imx/CMakeLists.txt

```diff
@@ -10,8 +10,13 @@ zephyr_library_compile_definitions_ifdef(
   CONFIG_PTP_CLOCK_MCUX ENET_ENHANCEDBUFFERDESCRIPTOR_MODE
 )
 
+zephyr_library_compile_definitions_ifdef(
+  CONFIG_DEVICE_CONFIGURATION_DATA FSL_SDK_ENABLE_DRIVER_CACHE_CONTROL
+)
```

Comment:
I think this should be enabled unconditionaly, so that OCRAM is also supported. The impact with DTCM should be minimal.